docs/spfx/sharepoint-subscription-edition-component.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+description: SharePoint Server Subscription Edition (SE) supports to ungrade individual components within SharePoint Framework when developing client-side web parts.
+ms.date: 09/08/2023
+ms.localizationpriority: high
+---
+
+
+# Upgrade components in SharePoint Framework development with SharePoint Server Subscription Edition
+
+SharePoint Server Subscription Edition supports SharePoint Framework client-side web parts in classic and modern pages, and extensions in modern pages.
+
+When developing a client-side web part, you can only use the specific versions of components supported by the SharePoint Framework to make the web part compatible with your SharePoint Server.
+
+To use more modern versions of the SharePoint Framework (SPFx) components, this article introduces some ways to upgrade individual components without upgrading the major version of SPFx.
+
+> [!NOTE]
+> Because the component-wide upgrades were not made as part of a broader update to SPFx, some manual configuration of the development environment might be completed in order to utilize them.
+
+> [!IMPORTANT]
+> Only SharePoint Server Subscription Edition (SE) supports the component-wide upgrades for the SPFx development since the [Version 23H2](/sharepoint/what-s-new/new-and-improved-features-in-sharepoint-server-subscription-edition-23h2-release#sharepoint-framework-spfx-component-upgrades) feature update. SharePoint Server 2019 and older SharePoint Server products do not support that.
+
+## Upgrade React and Fluent UI React
+
+The SharePoint Framework v1.4.1 natively supports React v15.8 and Fluent UI React ([formerly Office UI Fabric React](https://devblogs.microsoft.com/microsoft365dev/ui-fabric-is-evolving-into-fluent-ui/)) v5.17.

```

Comment:
This assumes the reader is on SP SE GA... SP SE 23H1 updated SPFx to v1.5.1... should that be incorporated in this statement?